core/modules/wsched/BlendScheduler.h

```diff
@@ -119,13 +119,14 @@ class BlendScheduler : public wsched::SchedulerBase {
     bool ready() override;
     int applyAvailableThreads(int tempMax) override { return tempMax;} //< does nothing
 
-    void setFlagReorderScans() { _flagReorderScans = true; }
     int calcAvailableTheads();
 
     bool isScanSnail(SchedulerBase::Ptr const& scan);
     int moveUserQueryToSnail(QueryId qId, SchedulerBase::Ptr const& source);
     int moveUserQuery(QueryId qId, SchedulerBase::Ptr const& source, SchedulerBase::Ptr const& destination);
 
+    void setPrioritizeByInFlight(bool val) { _prioritizeByInFlight = val; }
```

Comment:
Is this thread safe?

Reply:
I made it atomic, but it's really only meant to be called at startup where it may for systems with limited memory.